tests/test_environment_variables.py

```diff
@@ -43,28 +43,43 @@ def test_parse_environment_variables_empty(monkeypatch):
             {},
         ),
         (
-            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": 0},
+            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": "0"},
             {},
         ),
         (
-            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": 1},
-            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": 1},
+            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": "1"},
+            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": "1"},
         ),
         (
             {"CONVERT2RHEL_CONFIGURE_HOST_METERING": "auto"},
             {"CONVERT2RHEL_CONFIGURE_HOST_METERING": "auto"},
         ),
         (
-            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": 0, "SCRIPT_MODE": "ANALYSIS"},
+            {"CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": "0", "SCRIPT_MODE": "ANALYSIS"},
             {"SCRIPT_MODE": "ANALYSIS"},
         ),
         (
             {
-                "CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP": 1,
-                "CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": 0,
+                "CONVERT2RHEL_OUTDATED_PACKAGE_CHECK_SKIP": "1",
+                "CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS": "0",
```

Comment:
Is there also test for the "None" value? I assume now the Paramus should take everything except falsy values and "None"

Reply:
Currently, not... We expect that it should be 0 or 1. Not sure if that's needed to add a case for None values for those. 

I guess that advisor-backend is enforcing the value to be either 0 or 1, so anything that is not in that rage should be discarded.